roles/ocluster/tasks/main.yml

```diff
@@ -13,25 +13,40 @@
 - name: Stop ocluster via launchctl
   shell: launchctl unload ~/Library/LaunchAgents/com.tarides.ocluster.worker.plist
 
+- name: Remove ~/ocluster
+  file:
+    path: "{{ ansible_env.HOME }}/ocluster"
+    state: absent
+
 - name: Download the source from GitHub
   git:
     repo: https://github.com/ocurrent/ocluster.git
     dest: "{{ ansible_env.HOME }}/ocluster"
     force: yes
 
-- name: Checkout the correct SHA
-  shell: git fetch origin master && git checkout fcadac0bbd967495cb0ed17a6a2c722e333e4f3a --recurse-submodules
-  args: 
+- name: Switch to Tim's PR
+  shell: git fetch origin pull/202/head:test-branch && git checkout test-branch
```

Comment:
This can be changed to commit https://github.com/ocurrent/ocluster/commit/f2a94b163f763d57a4a63614a62b39c1020e53e7